examples/server_client.rs

```diff
@@ -2,45 +2,43 @@
 //! Technically, they both work the same.
 //! Note that in practice you don't want to implement a chat client using UDP.
 use std::io::stdin;
+use std::time::Instant;
 
 use laminar::{ErrorKind, Packet, Socket, SocketEvent};
-use std::thread;
 
 const SERVER: &str = "127.0.0.1:12351";
 
 fn test() {}
 
 fn server() -> Result<(), ErrorKind> {
-    let (mut socket, packet_sender, event_receiver) = Socket::bind(SERVER)?;
-    let _thread = thread::spawn(move || socket.start_polling());
```

Comment:
We do probably want to keep at least one example that uses the polling loop.

Reply:
Done